packages/core/src/tracing/instrumentation/protocols/grpcJs.js

```diff
@@ -112,8 +114,9 @@ function modifyArgs(name, originalArgs, span) {
 
       if (err) {
         const errorMessage = err.details || err.message;
-
-        if (errorMessage !== 'Cancelled') {
+        if (typeof errorMessage === 'string' && errorMessage.toLowerCase().includes('cancelled')) {
```

Comment:
I noticed that the tests had all the code in place to test cancelled calls (because they are to some degree a copy of the old grpc tests), but there were actually no tests for cancelling. When adding them, I found out that we need to tweak the code a bit to correctly detect cancelled calls.